lib/bamboo/adapters/mailgun_adapter.ex

```diff
@@ -123,7 +130,7 @@ defmodule Bamboo.MailgunAdapter do
   defp put_text_body(body, %Email{text_body: text_body}), do: Map.put(body, :text, text_body)
 
   defp full_uri(config) do
-    (Application.get_env(:bamboo, :mailgun_base_uri) || @base_uri)
-    <> config.domain <> "/messages"
+    [Application.get_env(:bamboo, :mailgun_base_uri) || @base_uri,
```

Comment:
It seems strange that this return a list and not a string. Is that just a Hackney thing?